presto-main/src/test/java/io/prestosql/sql/planner/TestDynamicFilter.java

```diff
@@ -72,6 +72,7 @@ public void testNonInnerJoin()
                         join(
```

Comment:
could we enhance `io.prestosql.sql.planner.optimizations.TestRemoveUnsupportedDynamicFilters` tests as part of this PR. Currently, these tests don't validate that joins no longer have DF.

Reply:
done